python/lsst/sdm_schemas/schemas/ivoa/obscore_nominal.yaml

```diff
@@ -17,6 +17,9 @@ tables:
     tap:std: 1
     tap:principal: 1
     ivoa:unit:
+    datatype: string
+    votable:arraysize: "*"
```

Comment:
Is the idea that because of some other code changes since I made this PR I can delete all the votable:arraysize lines?